src/app/moods/info/page.tsx

```diff
@@ -1,3 +1,164 @@
-export default function Page() {
-  return <h1>this is a placeholder</h1>;
+"use client";
+
+import Image from "next/image";
+import Link from "next/link";
+import { Header } from "@/ui/shared/Header";
+import Toggle from "@/ui/shared/Toggle";
+import { useState } from "react";
+
+const tasks = [
+  {
+    title: "Urgent & Important",
+    description: "Do these tasks immediately (e.g., deadlines, emergencies).",
+  },
+  {
+    title: "Not Urgent but Important",
+    description: "Schedule these tasks (e.g., planning, exercise).",
+  },
+  {
+    title: "Urgent but Not Important",
+    description: "Delegate these tasks if possible (e.g., interruptions).",
+  },
+  {
+    title: "Not Urgent & Not Important",
+    description: "Eliminate or minimize these tasks (e.g., distractions).",
+  },
+];
+
+const neuroInfo = [
+  {
+    title: "Dopamine",
+    description: "(Motivation)",
+  },
+  {
+    title: "Norepinephrine",
+    description: "(Intensity)",
+  },
+  {
+    title: "Serotonin",
+    description: "Emotional stability",
+  },
+];
+
+export default function MoodsInfoPage() {
+  const [isPriority, setIsPriority] = useState(false);
+
+  return (
+    <>
+      <Header title="Learn More" isInfoPage={true} />
+      <Toggle
+        isToggled={isPriority}
+        setIsToggled={setIsPriority}
+        toggledOff={"Mood"}
+        toggledOn={"Priority"}
+        showLabels={true}
+      />
+      <div className="w-10/12 m-auto flex flex-col gap-10 mt-8">
+        {!isPriority ? (
+          <>
+            <Image
+              src="/images/priorityCubeImage.png"
+              width={600}
+              height={600}
+              alt="Image of an Eisenhower Matrix"
+              layout="responsive"
+            />
+            <div className="flex flex-col gap-3">
+              <h2 className="text-xl font-semibold">The Eisenhower Matrix</h2>
+              <p>
+                The Eisenhower Matrix helps you prioritize tasks based on
+                urgency and importance. It divides tasks into four quadrants:
+              </p>
+            </div>
```

Comment:
```suggestion
            <div className="flex flex-col gap-3">
              <h2 className="text-xl font-semibold">The Eisenhower Matrix</h2>
              <p>
                The Eisenhower Matrix helps you prioritize tasks based on
                urgency and importance. It divides tasks into four quadrants:
              </p>
            </div>
            <Image
              src="/images/priorityCubeImage.png"
              width={600}
              height={600}
              alt="Image of an Eisenhower Matrix"
              layout="responsive"
            />
```
To be more consistent with with mood info page

Reply:
It is currently in line with the figma design, I propose we leave it that way for now